packages/vsf-storyblok-extension/index.js

```diff
@@ -69,5 +70,25 @@ module.exports = ({ config, db }) => {
       error: false
     })
   })
+
+  api.get('/validate-editor', async (req, res) => {
+    const { spaceId, timestamp, token } = req.query
+
+    const validationString = spaceId + ':' + config.storyblok.previewToken + ':' + timestamp
```

Comment:
Do we wanna use `${spaceId}:${config.storyblok.previewToken}:${timestamp}` instead?